application.js

```diff
@@ -1,8 +1,9 @@
 var updateSubtotal = function(ele){
-  var itemPrice = parseFloat($(ele).find('.item-price').text());
+  var itemPrice = parseFloat($(ele).find('.item-price').text().trim().slice(1));
```

Comment:
Because your HTML has a lot of space, the text() comes back with a lot of space. So you need to trim the whitespace, then remove the $ sign, before parseFloat.